internal/recert/recert.go

```diff
@@ -145,6 +145,11 @@ func CreateRecertConfigFile(seedReconfig *seedreconfig.SeedReconfiguration, seed
 		fmt.Sprintf("api-int.%s,api-int.%s", seedFullDomain, clusterFullDomain),
 		fmt.Sprintf("*.apps.%s,*.apps.%s", seedFullDomain, clusterFullDomain),
 	}
+	// check if there is an ingress CN provided for backwards compatibility
```

Comment:
Can we move the entire `config.CNSanReplaceRules = []string{` assignment to be *after* the `config.UseKeyRules = []string{` assignment to emphasize that internally in recert the former happens only after the latter? As in, the use-key-rule needs to use the name before the replacement, not after